infra/build/developer-tools/build/scripts/task_helper_functions.sh

```diff
@@ -562,6 +562,56 @@ finish_integration() {
   exit "${rv}"
 }
 
+
+# This function is called by /usr/local/bin/test_integration.sh and can be
+# overridden on a per-module basis to implement additional steps.
+run_terraform_validator() {
+  source_test_env
+
+  init_credentials
+  tf_name=$1
```

Comment:
Can we just use `base(tf_full_path)` as `tf_name`?
If `tf_full_path` is `/workdir/test/fixture/foo` It can then be stored in `test/integration/tmp/tfvt/foo/plan.json`